app/views/characters/index.haml

```diff
@@ -46,8 +54,9 @@
         = hidden_field_tag :search_screenname, true
         = hidden_field_tag :search_nickname, true
         = submit_tag "Search", class: 'button'
-  %tr
-    %th.odd{style: 'padding: 3px', colspan: colspan}
+  - if character_split == 'template' # spacer between two .subbers
+    %tr
+      %th.odd{style: 'padding: 3px', colspan: colspan}
```

Comment:
...huh. was that getting included even on non-split? should it not be?
also should possibly go in the... regular 'if view split' right below it

Reply:
it should not be, as the comment says, in both views there's a search bar at the top with .subber but only in template view would this then immediately be followed by a template name header _also_ with .subber which was confusing and so the spacer separates them; non-split doesn't have the template .subber sections so it doesn't need the spacer.